client/src/components/FilterOptions.jsx

```diff
@@ -0,0 +1,18 @@
+import { Status } from "../utils/utils";
+
+const FilterOptions = ( {filterEvents}) => {
```

Comment:
same as previous comment. It is not obvious that `filterEvents` is a callback function.
```suggestion
const FilterOptions = ( {onFilterChange}) => {
```